doc/services/device_mgmt/smp_protocol.rst

```diff
@@ -28,7 +28,7 @@ one byte lone, and takes the following form:
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
     |7|6|5|4|3|2|1|0|7|6|5|4|3|2|1|0|7|6|5|4|3|2|1|0|7|6|5|4|3|2|1|0|
     +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
-    |   Res   | OP  |      Flags    |          Data Length          |
+    |  Res  |V| OP  |      Flags    |          Data Length          |
```

Comment:
Don't you want to just take over all the `res` bits as version?

Reply:
We are not using all flags yet as far as I can tell. 
IMHO taking at least two bits for version would give more flexibility for future expansion, but will not insist.